contracts/Entity.sol

```diff
@@ -8,8 +8,6 @@ contract Entity is Controller, Proxy {
   constructor (address _settings, address _entityAdmin, bytes32 _entityContext) Controller(_settings) Proxy() public {
     _setDelegateAddress(settings().getRootAddress(SETTING_ENTITY_DELEGATE));
 
-    dataUint256["treasuryCollRatioBP"] = 10000;
-
     if (_entityContext != "") {
```

Comment:
I'm thinking if we should do a length check here instead?

`if(_entityContext.length != 0)`

Reply:
It's ` bytes32` so it's either going to be 32 bytes or nothing -> https://ethereum.stackexchange.com/questions/27227/how-to-check-if-a-bytes32-represents-an-empty-string

Reply:
Right. got you